### DISCUSSION TITLE
【求助】无法正常进入绳网

### DISCUSSION THREAD

**Discussion by zzyuJPG** [share121/inter-knot#3020] [崩坏：星穹铁道]
我于昨日晚上注册绳网账号，成功进入绳网，浏览数篇帖子，今日上午欲重入绳网与诸位同乐，却发现无法进入绳网，右上角时常弹出“获取用户信息失败”“获取文章失败”等文字，不知诸位是否否有办法助我返回绳网
![screenshot-1724041104297](https://github.com/user-attachments/assets/5ef96653-7011-4c21-9799-cc4ebc52ceef)
![screenshot-1724041104297](https://github.com/user-attachments/assets/1917a0e2-9a08-4a44-b2c8-0bf0bc17abde)
感激不尽

**Comment by 502awa** [comment]
无法进入绳网却在绳网发布求助
![E57E8E74DAA1CF54BBB574818BFC2747](https://github.com/user-attachments/assets/692df24a-80d2-4318-b11e-727d0c717470)

**Reply by zzyuJPG** [reply]
我直接用Github发的

**Comment by Nie7bai** [comment]
可以看看梯子是不是正常

**Reply by zzyuJPG** [reply]
用的steam++可以上Github但是进不了绳网

**Comment by FRIKY-c** [comment]
重装绳网插件即可

**Reply by zzyuJPG** [reply]
已经尝试过仍然无效，但是非常感谢能给出建议

**Comment by zzyuJPG** [comment]
承蒙各位厚爱，目前问题已通过换为火狐浏览器解决

**Reply by jsishsjsj** [reply]
这个问题在[121的视频中给出了解决办法](https://www.bilibili.com/video/BV1MPWAe3E23/)最好还是看一下吧

**Reply by zzyuJPG** [reply]
观看过后已经通过视频方法解决，十分感谢

**Comment by hk0852** [comment]
你会不会是在看其他帖子的时候直接把绳网的原网页叉掉了吧，虽然你从帖子返回了绳网，但不是原来的那个网页所以就导致了这个问题

**Comment by MTAlexKen** [comment]
![Screenshot_2024-08-19-17-39-08-714_mark via](https://github.com/user-attachments/assets/42f88424-b2d5-4fd9-8a81-d8a20522d0de)
我手机上用Via浏览器更新完1.14.1助手脚本也复现了，清数据+重新登录Github账号即可

**Reply by zzyuJPG** [reply]
感谢解答，方法很好用，目前已恢复正常，谢谢
